telemetry/telemetrySchema.json

```
@@ -65,6 +65,10 @@
                     "passive": {
                         "type": "boolean",
                         "description": "Set as true if the metric is emitted without any explicit user action."
+                    },
+                    "trackPerformance": {
+                        "type": "boolean",
+                        "description": "Set as true if the metric can track run-time performance information."
```

should we mention "defaults to false" ?
```suggestion
                        "description": "Set as true if the metric can track run-time performance information. Default=false"
```